### PR TITLE
[UI:Submission] Mobile view for submission page

### PR DESCRIPTION
**Pull request by mukul-kmr-jha**
### Please check if the PR fulfills these requirements:

* [x] The PR title and message follows our [guidelines](https://submitty.org/developer/how_to_contribute)
* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### What is the new behavior?
Makes the submission page mobile-compatible with all the subsections with single or multiple dropzones, TA results, version selections, submitted-files-info-section, and grade inquiry posts.

### Other information?
- ~working on screen where we have more than 1 dropzone~ 
- Notebooks works well but @shailpatels is looking into some design changes/refactor for notebooks.

### CODE REVIEW

File: site/app/templates/submission/homework/CurrentVersionResults.twig [hunk 1]
```diff
@@ -8,66 +8,103 @@
...
             {% if results is defined %}
-                grading time: {{ grade_time }} seconds<br />
+                <span class="flex-row">
+                    <span id="submission_timestamp">Grading time:</span>
+                    <span>{{ grade_time }} seconds</span>
+                </span>
                 {% if num_autogrades == 1 %}
-                    queue wait time: {{ wait_time }} seconds<br />
+                    <span class="flex-row">
+                        <span id="submission_timestamp">Queue wait time:</span>
```
Comment (elihschiff) [inline site/app/templates/submission/homework/CurrentVersionResults.twig:58]: There are a bunch of element all with the same id `submission_timestamp`
You might want to give them all a unique id, or switch `submission_timestamp` to be a class if it is used for css

Comment (mukul-kmr-jha) [reply site/app/templates/submission/homework/CurrentVersionResults.twig]: added, thanks!

**Comment by bmcutler** [comment]
Minor point, but the "display whitespace/non-printing characters as escape sequences" button is too wide for mobile.  NOTE: I'm  just testing this on my laptop, I'm not sure of the display resolution for mobile.  I think this button text should wrap instead.  

whitespace


And also, the text output for autograding tests is often wide and requires scrolling horizontally.  We do NOT want to wrap this.   It does scroll ok, and I'm not sure how best to style this, but it looks accidental how it overlaps the div boxes.  Maybe we can discuss options as a group in our meeting today?

wide_autograding


screenshots from  "Autograde and TA Homework (C System Calls)"

**Comment by mukul-kmr-jha** [comment]
Added the changes, but there is a small merge conflict over the changes in the submitted-files info box 

Group 4
Group 3

please take a look and add suggestions, after that I will push the code without any conflicts. Thanks!
@shailpatels

**Comment by bmcutler** [comment]
@mukul-kmr-jha 
I prefer the final image.
Barb

**Comment by mukul-kmr-jha** [comment]
Done, please take another look... thanks!

**Pull request merged**